packages/library-export-default-webpack-plugin/index.js

```diff
@@ -16,7 +15,7 @@ module.exports = class LibraryExportDefaultPlugin {
 				const { mainTemplate, chunkTemplate } = compilation;
 
 				const onRenderWithEntry = ( source, chunk ) => {
-					if ( ! includes( this.entryPointNames, chunk.name ) ) {
+					if ( ! this.entryPointNames.includes( chunk.name ) ) {
```

Comment:
Given that `this.entryPointNames` is initialised to a value passed into the constructor, it may be worth it being a bit defensive here and doing e.g.:

```
if ( ! (this.entryPointNames || []).includes( chunk.name ) ) {
```

Reply:
Or fix the `this.entryPointNames` assignment in the constructor 🙂

Reply:
Good point, updated it to default to an empty array in https://github.com/WordPress/gutenberg/pull/44571/commits/601eb35e134ef88ec6cf4740798008a6b1d1ff6b